File: rev-com-server/src/main/java/com/codergb/bean/book/BookCate.java
package com.codergb.bean.book;

import lombok.*;

import java.util.Date;
import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class BookCate {
  private String id;
  private String name;
  private List<BookCateCh> bookCateChList;
  private Date createTime;
  private Date updateTime;
}

File: rev-com-manage/src/types/book/bookCate.ts
interface IBookCateCh {
  id: string;
  name: string;
  createTime: string;
  updateTime: string;
}
interface IBookCate {
  id: string;
  name: string;
  bookCateChList: IBookCateCh[];
  createTime: string;
  updateTime: string;
}
export type { IBookCateCh, IBookCate };
